gsa/src/web/pages/tasks/details.js

```diff
@@ -94,15 +94,14 @@ class TaskDetails extends React.Component {
       inAssets,
       reports,
       minQod,
-      preferences,
       scanner,
       target,
       maxChecks,
       maxHosts,
       schedule,
+      sourceIface = '',
```

Comment:
Could we rename that variable to `sourceInterface` in another PR? also in hyperion?

Reply:
of course. when i first saw that variable i was a little confused too.